packages/checkbox/src/Check.tsx

```diff
@@ -150,45 +151,44 @@ export function Check({
   selector,
 }: CheckProps) {
   const { usingKeyboard } = useUsingKeyboardContext();
-  const baseFontSize = useUpdatedBaseFontSize();
+  const prefersReducedMotion = usePrefersReducedMotion();
 
   const CheckIcon = indeterminate ? SvgIndeterminate : SvgCheck;
   const showCheckIcon = indeterminate || isChecked;
   const checkIconColor = disabled ? palette.gray.light3 : palette.white;
-  const shouldAnimate = animate && !indeterminate;
+  const shouldAnimate = animate && !indeterminate && !prefersReducedMotion;
 
   return (
     <>
       <div
-        className={cx(
-          selector,
-          checkWrapperBaseStyle,
-          checkWrapperAlignment[baseFontSize],
-          {
-            [checkWrapperCheckedStyle]: showCheckIcon,
-            [checkWrapperDisabledStyle]: disabled,
-            [checkWrapperCheckedDisabledStyle]: disabled && showCheckIcon,
-          },
-        )}
+        className={cx(selector, checkWrapperBaseStyle, {
+          [checkWrapperCheckedStyle]: showCheckIcon,
+          [checkWrapperDisabledStyle]: disabled,
+          [checkWrapperCheckedDisabledStyle]: disabled && showCheckIcon,
+          [disableAnimation]: !shouldAnimate,
+        })}
       >
         <Transition
           in={showCheckIcon}
-          timeout={checkAnimationDuration}
+          timeout={prefersReducedMotion ? 0 : checkAnimationDuration}
```

Comment:
Can you explain what this is doing? I set it to 0 and I didn't see a difference.

Reply:
If the user has the `prefers-reduced-motion` preference set, then the transition timeout will be 0. This needs to be the same value as the `transition-duration` property in css

Reply:
I see, however i don't visually see any difference in the animation if `shouldAnimate` is `false` and the `timeout` is `100 or 0` or if `shouldAnimate` is `true` and `timeout` is `100 or 0`.  I can see that when it's `false` it doesn't animate and if it `true` it does. I don't think this is blocking, i'm just curious because I can't see a difference.

Reply:
Ah I see. I guess this could just stay as it was before and it'd behave the same. I think it's more "technically correct" as written now even if not entirely necessary